DOMAINS_LINK.txt

```diff
@@ -1 +1,4 @@
 https://github.com/GSA/data/tree/gh-pages/dotgov-domains
+
+Current Federal Data from:
+https://raw.githubusercontent.com/GSA/data/gh-pages/dotgov-domains/current-federal.csv
```

Comment:
Yea that's a good point - and yes they are just my own notes.

I will make a PR for that next